Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml

```diff
@@ -144,3 +144,15 @@
       coefficients:
         Heat: 0.95
   - type: GroupExamine
+
+- type: entity
+  parent: ClothingEyesBase
+  id: ClothingEyesVisorNinja
+  name: ninja visor
+  description: An advanced visor protecting the ninja's eyes from flashing lights.
```

Comment:
When looking at an object, perhaps should be "a ninja's eyes" because a Sec officer might have found them for instance.